src/utils/index.js

```diff
@@ -54,7 +54,7 @@ export const formatDecimal = (decimal, significantDecimalPlaces = 2) => {
 };
 
 export const getBtcFromSatoshis = (satoshis) =>
-    satoshis.dividedBy(100000000).toNumber();
+    satoshis.dividedBy(100000000).toString();
```

Comment:
Probably better to usa a `toFixed`, so that the whole number is taken as-is, with all the decimals and the maximum precision.